### PR TITLE
testTC_017_015_сreateAPipeline

### PR DESCRIPTION
**Pull request by helenvel**
testTC_017_015_сreateAPipeline

### CODE REVIEW

File: src/test/java/HelenVelJenkinsTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @Test
+    public void testTC_017_015_сreateAPipeline() {
+
+        getDriver().findElement(By.xpath(newItemButton)).click();
+        getDriver().findElement(By.xpath(itemNameField)).sendKeys("Pipeline");
+        getDriver().findElement(By.xpath(itemPipeline)).click();
+        getDriver().findElement(By.xpath(itemButtonOk)).click();
+        getDriver().findElement(By.xpath(itemButtonSave)).click();
+        getDriver().findElement(By.xpath(itemDashboardClick)).click();
+        WebElement jobItem = getDriver().findElement(By.xpath(dashboardName));
```
Comment (RusAli) [inline src/test/java/HelenVelJenkinsTest.java:27]: it will be great to add white space between 26 and 27 rows to divide logic

Comment (helenvel) [reply src/test/java/HelenVelJenkinsTest.java:27]: Спасибо, добавлю

**Review by anonymoustuna — APPROVED** [review]

**Comment by helenvel** [comment]
https://trello.com/c/8bCUUSOS/441-at017015-pipeline-log-in-to-jenkins-create-a-pipeline-make-sure-that-the-pipeline-is-created-and-displayed-on-the-dashboard

**Comment by helenvel** [comment]
[https://trello.com/c/8bCUUSOS/441-at017015-pipeline-log-in-to-jenkins-create-a-pipeline-make-sure-that-the-pipeline-is-created-and-displayed-on-the-dashboard](url)

**Pull request merged**